app/jobs/build_tweets_job.rb

```diff
@@ -0,0 +1,10 @@
+class BuildTweetsJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(*args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `perform(*)` if you want the method to accept any arguments but don't care about them.